vpr/src/draw/draw_types.h

```diff
@@ -193,6 +194,7 @@ struct t_draw_state {
     bool forced_pause = false;
     int sequence_number = 0;
     float net_alpha = 0.1;
+    float pres_fac = 1.;
```

Comment:
We'll have to be careful about how this is kept up to date. It would be really difficult to debug if this ever got out of sync with the router pres_fac as the graphics would be showing something different.